client/src/components/Pages/Structure/RegElement/RegElementForm/RegElementForm.tsx

```diff
@@ -31,23 +39,52 @@ interface Props {
   regulatoryClassItems: object;
   elementClass: RegulatoryClass | "default";
   setElementClass: CallableFunction;
+  featureId: string;
+  setFeatureId: CallableFunction;
   gene: string;
   setGene: CallableFunction;
   geneText: string;
   setGeneText: CallableFunction;
+  chromosome: string;
+  setChromosome: CallableFunction;
+  genomicStart: string;
+  setGenomicStart: Setter<string>;
+  genomicEnd: string;
+  setGenomicEnd: Setter<string>;
 }
 
 const RegElementForm: React.FC<Props> = ({
   regulatoryClassItems,
   elementClass,
   setElementClass,
+  featureId,
+  setFeatureId,
   gene,
   setGene,
   geneText,
   setGeneText,
+  chromosome,
+  setChromosome,
+  genomicStart,
+  setGenomicStart,
+  genomicEnd,
+  setGenomicEnd,
 }) => {
   const classes = useStyles();
 
+  const [txStartingGenomicText, setTxStartingGenomicText] = useState("");
+  const [txEndingGenomicText, setTxEndingGenomicText] = useState("");
+
+  const inputComplete = gene === "";
+  const validated = inputComplete;
+  const [, setExpanded] = useState<boolean>(!validated);
```

Comment:
is this basically just getting used to force a rerender?

Reply:
Added in #373